runner/app/live/pipelines/streamdiffusion.py

```diff
@@ -343,15 +341,15 @@ def _prepare_controlnet_configs(params: StreamDiffusionParams) -> Optional[List[
     return controlnet_configs
 
 
-def load_streamdiffusion_sync(params: StreamDiffusionParams, engine_dir = "engines", build_engines_if_missing = False):
+def load_streamdiffusion_sync(params: StreamDiffusionParams, min_batch_size: int, max_batch_size: int, engine_dir = "engines", build_engines_if_missing = False):
```

Comment:
I think we need the default here as well since we don't send it on this pipeline
```suggestion
def load_streamdiffusion_sync(params: StreamDiffusionParams, min_batch_size = 1, max_batch_size = 4, engine_dir = "engines", build_engines_if_missing = False):
```